### ISSUE TITLE
Fails to clone/update private repositories

### ISSUE THREAD

**Issue by cronik** [getantibody/antibody#329]
After upgrading to 4.3.0 cloning/updating repositories that require auth no longer work. 

```
$ antibody bundle < ~/.zsh_plugins.txt                    
antibody: git clone failed for https://bitbucket.example.com/scm/foo/bar.git Cloning into '/Users/billybob/Library/Caches/antibody/https-COLON--SLASH--SLASH-bitbucket.example.com-SLASH-scm-SLASH-foo-SLASH-bar.git'...
error: cannot run 0: No such file or directory
fatal: could not read Username for 'https://bitbucket.example.com': terminal prompts disabled
```

This seems to be due to changes introduced by #325

**Comment by caarlos0** [comment]
hmm how did you authenticate before?

**Comment by cronik** [comment]
I use a mac so it uses the default "osxkeychain" store.

**Comment by caarlos0** [comment]
can you check if it works on master?

if you don't have tools to compile and etc, let me know your platform and I'll attach it here

**Comment by cronik** [comment]
built from master and it works as expected, thanks!

**Comment by caarlos0** [comment]
will release a new version, thanks for pointing this out and testing 🙏

**Comment by caarlos0** [comment]
fixed in v4.3.1
